src/main/resources/assets/securitycraft/lang/pt_br.json

```diff
@@ -0,0 +1,1198 @@
+	"block.securitycraft.secret_mangrove_sign": "Sinal de mangue secreto",
+	"block.securitycraft.secret_crimson_sign": "Sinal carmesim secreto",
+	"block.securitycraft.secret_warped_sign": "Sinal de distorção secreta",
+	"block.securitycraft.reinforced_observer": "Observador reforçado",
+	"block.securitycraft.gold_mine": "Mina de minério de ouro",
+	"block.securitycraft.iron_mine": "Minério de ferro",
+	"block.securitycraft.coal_mine": "Minério de carvão mina",
+	"block.securitycraft.nether_gold_mine": "Mina de minério de ouro inferior",
+	"block.securitycraft.lapis_mine": "lapisLazuliOreMine",
+	"block.securitycraft.redstone_mine": "Redstone Ore Mine",
+	"block.securitycraft.emerald_mine": "Mina de minério de esmeralda",
+	"block.securitycraft.quartz_mine": "Mina de minério de quartzo inferior",
+	"block.securitycraft.gilded_blackstone_mine": "Mina Blackstone dourada",
+	"block.securitycraft.ancient_debris_mine": "Mina de detritos antigos",
+	"block.securitycraft.reinforced_redstone_lamp": "Lâmpada Redstone reforçada",
+	"block.securitycraft.reinforced_hopper": "Tremonha reforçada",
+	"block.securitycraft.keypad_door": "Porta do teclado",
+	"block.securitycraft.reinforced_cauldron": "Caldeirão reforçado",
+	"block.securitycraft.reinforced_water_cauldron": "Caldeirão de água reforçada",
+	"block.securitycraft.reinforced_lava_cauldron": "Caldeirão de lava reforçado",
+	"block.securitycraft.reinforced_powder_snow_cauldron": "Caldeirão de neve em pó reforçado",
+	"block.securitycraft.reinforced_piston": "Pistão reforçado",
+	"block.securitycraft.reinforced_sticky_piston": "Pistão pegajoso reforçado",
+	"block.securitycraft.reinforced_piston_head": "Cabeça de pistão reforçado",
+	"block.securitycraft.reinforced_moving_piston": "Pistão em movimento reforçado",
+	"block.securitycraft.copper_mine": "Mina de minério de cobre",
+	"block.securitycraft.deepslate_mine": "DeepSlate Mine",
+	"block.securitycraft.cobbled_deepslate_mine": "DeepLate Mine",
+	"block.securitycraft.deepslate_copper_mine": "Mina de minério de cobre DeepSlate",
+	"block.securitycraft.deepslate_coal_mine": "Mina de minério de carvão DeepSlate",
+	"block.securitycraft.deepslate_iron_mine": "Mina de minério de ferro DeepSlate",
+	"block.securitycraft.deepslate_gold_mine": "Mina de minério de ouro de profundidade",
+	"block.securitycraft.deepslate_redstone_mine": "Mina de minério de Redstone DeepSlate",
+	"block.securitycraft.deepslate_emerald_mine": "deepslateEmeraldOreMine",
+	"block.securitycraft.deepslate_lapis_mine": "Deepslate Lapis Lazuli Ore Mine",
+	"block.securitycraft.deepslate_diamond_mine": "Mina de minério de diamante DeepSlate",
+	"block.securitycraft.reinforced_deepslate": "Reforçado DeepSlate",
+	"block.securitycraft.reinforced_cobbled_deepslate": "Deepslate reforçado de paralelepípedos",
+	"block.securitycraft.reinforced_polished_deepslate": "DeepLate polido reforçado",
+	"block.securitycraft.reinforced_tuff": "Tuff reforçado",

```

Comment:
There are many reinforced blocks that don't match the vanilla name. I'll just use this one as an example: Vanilla's Tuff is called "Tufo" in brazilian portuguese, so the reinforced name would be like "Tufo reforçado". Further down, you translated "Reinforced Chiseled Deepslate" as "Deepslato esculpido reforçado", while the vanilla translation of "Chiseled Deepslate" is "Ardósia talhada". Please make sure all the reinforced names match vanilla's names.

Reply:
Okay